### PR TITLE
Custom API url option

### PR DESCRIPTION
**Pull request by bolchowka**
Ability to pass custom API address as an option for `WebAPI` and `RTMAPI` classes.

cc @kogierman @janowsiany

### CODE REVIEW

**Review by kogierman — COMMENTED** [review]
in general looks good, please just move `xxxAPIOptions` interfaces to `src/objects` (next to i.e. `RTMRequest`), `internal` is by design meant to be used only by sdk itself ;)

**Comment by bolchowka** [comment]
Sure, done ✅

File: src/agent/rtm.ts [hunk 1]
```diff
@@ -27,11 +27,11 @@ import type {
   Pushes,
 } from "./structures";
 import { ChatAccess, Event, Properties, RoutingStatus } from "../objects";
-import { RTMAPI } from "../internal/index";
+import { RTMAPI, RTMAPIOptions } from "../internal/index";
```
Comment (kogierman) [inline src/agent/rtm.ts:30]: `RTMAPIOptions` is not in internal anymore ;)

File: src/agent/web.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { WebAPI } from "../internal";
+import { WebAPI, WebAPIOptions } from "../internal";
```
Comment (kogierman) [inline src/agent/web.ts:1]: as above

File: src/configuration/index.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { WebAPI } from "../internal";
+import { WebAPI, WebAPIOptions } from "../internal";
```
Comment (kogierman) [inline src/configuration/index.ts:1]: as above

File: src/customer/rtm.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { RTMAPI } from "../internal";
+import { RTMAPI, RTMAPIOptions } from "../internal";
```
Comment (kogierman) [inline src/customer/rtm.ts:1]: as above

File: src/customer/web.ts [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import { WebAPI } from "../internal";
+import { WebAPI, WebAPIOptions } from "../internal";
```
Comment (kogierman) [inline src/customer/web.ts:1]: as above

**Comment by bolchowka** [comment]
@kogierman I've missed that, sorry! 🙈
I updated the code and `npm run build` now runs without errors. You'll see a few updates by prettier in my commit as well.

**Review by kogierman — APPROVED** [review]

**Review by kogierman — APPROVED** [review]

**Pull request merged**